hwy/contrib/unroller/unroller_test.cc

```diff
@@ -452,7 +452,8 @@ struct TestFind {
 
       FindUnit<T> cvtfn(ConvertScalarTo<T>(num - 1));
       MakeSigned<T> idx = 0;
-      Unroller(cvtfn, a, &idx, static_cast<ptrdiff_t>(num));
+      Unroller(cvtfn, const_cast<const T*>(a), &idx,
```

Comment:
No need for const_cast here, I think. It's always OK to take a const view of a non-const thing.

Reply:
Updated the test case to make it more clear.

I added that intentionally so test checks if const inputs are allowed.